authorization/views.py

```diff
@@ -2,6 +2,7 @@
 from django.contrib.auth.views import redirect_to_login
 from django.contrib.messages import error as error_message
 from django.core.exceptions import PermissionDenied
```

Comment:
```suggestion
```
`PermissionDenied`: this import is now unused, right? It can be then removed.

Reply:
Thanks for pointing out those issues, I changed the comment and removed the import. 

The render method has an optional argument for the status code, so I added a status=403 to the render call.

```
def render(request, template_name, context=None, content_type=None, status=None, using=None):
    """
    Return a HttpResponse whose content is filled with the result of calling
    django.template.loader.render_to_string() with the passed arguments.
    """
    content = loader.render_to_string(template_name, context, request, using=using)
    return HttpResponse(content, content_type, status)
```